lib/build.gradle

```diff
@@ -102,7 +103,9 @@ dependencies {
 
 	gsonCompileOnly 'com.google.code.gson:gson:2.10.1'
 
-	cleanthatCompileOnly 'io.github.solven-eu.cleanthat:java:2.0'
+	// TODO How can one add a test module like 'compatKtLint0Dot48Dot0'?
+	// cleanthatCompileAndTestOnly 'io.github.solven-eu.cleanthat:java:2.1'
```

Comment:
If you use the `versionCompatibility` block and do a `namespaces.register('cleanthat')` then you can follow the KtLint example.

Reply:
I still get `Could not find method cleanthatCompileAndTestOnly() for arguments [io.github.solven-eu.cleanthat:java:2.1] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.`